HLG/iocBoot/iocHLG-IOC-01/st-common.cmd

```diff
@@ -7,7 +7,7 @@ epicsEnvSet "STREAM_PROTOCOL_PATH" "$(HLG)/data"
 $(IFRECSIM) drvAsynSerialPortConfigure("L0", "$(PORT=NUL)", 0, 1, 0, 0)
 
 # For dev sim devices
-$(IFDEVSIM) drvAsynIPPortConfigure("L0", "localhost:$(EMULATOR_PORT=)")
+$(IFDEVSIM) drvAsynIPPortConfigure("L0", "localhost:$(EMULATOR_PORT=57677)")
```

Comment:
Did you really mean to change the HLG?

Reply:
Not sure how that IOC got put in my commit :/

Reply:
Removed that file.